api/swagger/swagger.yaml

```diff
@@ -161,6 +161,24 @@ paths:
         responses:
           "200":
             description: 200 Response
+  /samplingfeaturedatasets/:
+    get:
+      tags:
+      - samplingfeaturedatasets
```

Comment:
This tag `samplingfeaturedatasets` is not available, you have to specify this tag first up above.